### DISCUSSION TITLE
New LedgerSMB Logo Discussion

### DISCUSSION THREAD

**Discussion by neilt** [ledgersmb/LedgerSMB#7315] [Ideas]
I would like to propose a new LedgerSMB logo.

The reasons are:
1. The old logo is really dated.
2. A scalable version of the old logo source does not exist.
3. The gradient effects don't work well with transparent backgrounds.
4. The logo does not look good enlarged.

Attached is a proposal for discussion.  I am not a graphic designer and am not in any way wedded to this proposed version.

In fact, the proposal is not ready for prime time yet as it is just a quick mock-up.  The position of the elements is by sight only and they need to be positioned more accurately.  The overall size is probably not correct for web usage.  This mock-up was done using Inkscape and the original file is SVG, so anyone can edit it.  Since it is vector graphics it can be resized easily. I did not post the SVG file, but will if anyone wants to wack at it.

It uses the Helvetica Neue font because that is what the web site uses (at least as far as I can tell).  It uses the same basic colors because that is what the web site uses.  I think the colors are a bit dated, but that is a change for another time when or if the web site gets renovated.

Please leave comments.  I challenge you to hurt my feelings.  So go for it.

![NewLedgerSMBLogoHelveticaNeue](https://user-images.githubusercontent.com/1809796/230431576-961e2f91-89b8-48da-8383-585ed0484093.png)

**Comment by freelock** [comment]
I like the direction you're going, but don't like this logo. Feels a little cartoony to me -- but I think the contents are great.

I think it's the font for LedgerSMB that I don't like -- I do like the "Accounting, Inventory, ERP" line in there, and that part looks good.

Maybe play around with different fonts for "LedgerSMB"? and also gradients, shadows...

**Comment by neilt** [comment]
This initial proposal is using Helvetica Neue, which I thought was the same font as ledgersmb.org?

**Comment by neilt** [comment]
Here is the source.

![NewLedgerSMBLogo](https://user-images.githubusercontent.com/1809796/230477176-36df019f-4f6b-432b-b920-0605808674c6.svg)

**Comment by ehuelsmann** [comment]
summarizing comments in the chat channel:

"Signika Negative" also has skewed sides of the M, but is otherwise "lighter" than the Helvetica Neue font face above.

**Comment by neilt** [comment]
The SVG uses "Signika Negative" (the source).

**Comment by neilt** [comment]
Ahh, GitHub or my browser is substituting Fonts. That is why it does not look correct.

**Comment by ehuelsmann** [comment]
![2023-04-07_09-53](https://user-images.githubusercontent.com/2326559/230567681-fcf190dc-9701-4ba3-8364-b71e3e608bc2.png)

**Comment by neilt** [comment]
Here are the current sources in SVG format for both the logo and favicon, based on chat suggestions.

Safari is substituting fonts, so the SVG versions are not using the correct font.  The correct font is Signika Negative, normal style.  Brave and Firefox work correctly.

![favicon](https://user-images.githubusercontent.com/1809796/230628359-9d23925f-fb2e-41bb-9cdf-47297035e574.svg)
![NewLedgerSMBLogo](https://user-images.githubusercontent.com/1809796/230628439-17f218e3-f4e5-4c9b-aa9d-1651439cfff3.svg)

Here are the PNGs at 64x64 and 512x265 with the correct font.

![favicon8](https://user-images.githubusercontent.com/1809796/230629352-dc345b82-1ae4-41ec-bb88-3ac8911d959d.png)

![NewLedgerSMBLogo512x156](https://user-images.githubusercontent.com/1809796/230629333-dd29f397-e1b8-4a6d-b8ef-d53f7fc74b14.png)

**Comment by neilt** [comment]
@freelock suggested "the shadow on the letters seems a bit crisp, can that be fuzzier?" which has not been addressed

**Reply by ehuelsmann** [reply]
I know how to do that only with CSS: it's the third parameter in the CSS shadow which does that: the first is the vertical offset; the second the horizontal offset and the third is the "blur".

**Comment by neilt** [comment]
Attached is the final image at 512x256 px resolution, with background blur added.
![NewLedgerSMBLogo512x256](https://user-images.githubusercontent.com/1809796/230906082-7a557368-66df-47ba-9e68-b97d561b4290.png)

**Comment by ehuelsmann** [comment]
This has been merged to master and now replaced the old logo.
